dev/com.ibm.ws.security.token.ltpa_fat/fat/src/com/ibm/ws/security/token/ltpa/fat/LTPAKeyRotationTests.java

```diff
@@ -1602,7 +1825,7 @@ public static void updateConfigDynamically(LibertyServer server, ServerConfigura
         String logLine = server.waitForStringInLogUsingMark("CWWKG001[7-8]I");
 
         // Wait for feature update to be completed or LTPA configuration to get ready
-        Thread.sleep(200);
```

Comment:
we should try to remove this wait if possible, it should wait long enough once the right message is detected, like LTPA config ready